### ISSUE TITLE
Problems with the Freescout APP and Android 13

### ISSUE THREAD

**Issue by gplast1981** [freescout-helpdesk/freescout#3290]
I have problems with the Freescout APP and Android 13 here no login mask opens after entering the https address. It also doesn't work with https://demo.freescout.net/. The picture stays black.

With older Android and Iphone everything works fine. Why is that?

![grafik](https://github.com/freescout-helpdesk/freescout/assets/138209591/7d307ca4-f1d0-4468-b1f6-409de47dfc92)

![grafik](https://github.com/freescout-helpdesk/freescout/assets/138209591/5fc74cf6-2800-4529-b177-94650a030757)

**Comment by freescout-helpdesk** [comment]
Did it work for you on Android 13 before?

**Comment by gplast1981** [comment]
I have managed it, I have reset the Adroid phone to factory settings, then all updates and reinstalled Freescout now it works.

However, the push notifications do not work on Android 13, I bought the module Mobile Notification and installed it successfully on Iphone it works, with android 9 it works, but with Adroid 13 it does not work. Under Settings Notifications the app Freescout is not shown at all and under Systemapp it is shown but is denied for notifications. What is the reason ? I have installed everything NEW 

![IMG_6292](https://github.com/freescout-helpdesk/freescout/assets/138209591/537a99a3-9437-42f4-b93d-3fe087ed344e)

![IMG_6293](https://github.com/freescout-helpdesk/freescout/assets/138209591/518f869c-ca9b-40d6-9a22-2168b0e2d28a)
